src/block_cache/block_chunkcache.c

```diff
@@ -997,6 +1082,19 @@ __wt_chunkcache_teardown(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __wt_chunkcache_salvage --
+ *     Remove any knowledge of any extant chunk cache metadata. We can always rebuild the cache
+ *     later, so make no attempt at a "real" salvage.
+ */
+int
+__wt_chunkcache_salvage(WT_SESSION_IMPL *session)
+{
+    WT_RET_NOTFOUND_OK(__wt_metadata_remove(session, WT_CC_METAFILE_URI));
```

Comment:
It would be helpful to explain the distinction between a schema drop and simply removing an entry from metadata. Should we also delete the metadata file, or will `__wt_metadata_remove` take care of that?

Reply:
> It would be great if you could explain the distinction between a schema drop and simply removing an entry from metadata.

Do you mean in this thread, just to understand the PR? Or as a code comment?

> Should we also delete the metadata file, or will __wt_metadata_remove take care of that?

I don't believe there's any need to delete it. If we create the file again in future, the old one will get renamed, which seems reasonable if you've run salvage at some point previously.

Reply:
In this PR, thanks.

Reply:
Sure -  we don't need operation tracking or an internal session, so there was no point in having all of the machinery around a schema call. It also removed the complexity around taking the schema lock.